xlators/storage/posix/src/posix-inode-fd-ops.c

```diff
@@ -825,17 +825,19 @@ posix_do_fallocate(call_frame_t *frame, xlator_t *this, fd_t *fd, int32_t flags,
     return ret;
 }
 
-char *
-_page_aligned_alloc(size_t size, char **aligned_buf)
+static char *
+_page_aligned_alloc(size_t size, char **aligned_buf, gf_boolean_t calloc)
```

Comment:
Just in case you resend this patch, please change this name to something like zerofill or something.